a3p-integration/proposals/n:upgrade-next/package.json

```diff
@@ -16,10 +16,10 @@
       "inter-protocol/updatePriceFeeds.js price-feeds A3P_INTEGRATION",
       "vats/add-auction.js price-feeds",
       "vats/upgradeVaults.js price-feeds",
-      "inter-protocol/updatePriceFeeds.js submission/main main",
+      "inter-protocol/updatePriceFeeds.js submission/main MAINNET",
```

Comment:
Why is this even necessary?

Reply:
It is used to generate the artifacts that can be uploaded (via cosgov, e.g.) to make a proposal. @dckc has been a strong advocate that those artifacts should be produced in CI.

Reply:
Um ok but we're not deploying this with a core eval afaik?

Is there a specific reason the upgrade-next layer needs to produce these artifacts ?